contributed/check_properties_of_haloes_in_CosmoDC2.ipynb

```diff
@@ -0,0 +1,659 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#written by Marina Ricci\n",
+    "\n",
+    "#purposes: handling cosmoDC2 DESC catalog and investigating the properties of galaxy clusters \n",
+    "# (defined as halos with M_halo>1e14 Msol)\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Purpose of this notebook: \n",
+    "Objects in mock catalogs have to be representative of the their true population. In this context, this notebook aims at testing some basic statistical properties of galaxies belonging to clusters (arbitrarly defined here as halos with friend of friend mass > 10^14 Msol). The properties tested are : the mass/redshift distribution, the \"richness\"/mass relation, the density profile, the luminosity function and the color magnitude diagrams. These are the main intrinsic properties that drive cluster detection in the optical/NIR. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "import GCRCatalogs\n",
+    "from GCR import GCRQuery\n",
+    "from astropy.cosmology import FlatLambdaCDM\n",
+    "import astropy.units as u\n",
+    "from scipy.stats import normaltest\n",
+    "import matplotlib.colors as clr\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## - Handling the catalog"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# see all available catalogs\n",
+    "print(sorted(GCRCatalogs.get_available_catalogs(False)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load a calalog\n",
+    "catalog1 = GCRCatalogs.load_catalog('cosmoDC2_v1.1.4_small')\n",
+    "\n",
+    "#print catalog info\n",
+    "print(catalog1.get_catalog_info())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#define same cosmological parameters as the catalog\n",
+    "cosmo = FlatLambdaCDM(H0=71, Om0=0.2648, Ob0=0.0448) #not 100% sure it is the same cosmo as in the simu"
```

Comment:
I wanted to point out that you can get the cosmological parameters to check whether these are correct as follows:
```
catalog1.cosmology
```
This will tell you the cosmological parameters, so you can check and then clarify this line.  ([This tutorial](https://github.com/LSSTDESC/DC2-analysis/blob/master/tutorials/extragalactic_gcr_hod.ipynb) has a nice usage of the catalog-based cosmological parameter access, which is used to create a CCL cosmology object, if you're interested to see how this works.)